### PR TITLE
Create Reverse Nodes in k-Group

### PR DESCRIPTION
**Pull request by NidhiSingh26**
Hi @mr-ayush-agrawal  I have submitted 2 Leetcode solution of Hard level. Please check it out

### CODE REVIEW

**Comment by mr-ayush-agrawal** [comment]
Thanks @NidhiSingh26.
I have a suggestion that it would be great if you could add the link to the problem may be as a comment at the top would also work. That would be great for other while referring the code.

🥳 Happy Hacking

**Comment by NidhiSingh26** [comment]
@mr-ayush-agrawal  Sure

File: Leet Code/Reverse Nodes in k-Group [hunk 1]
```diff
@@ -0,0 +1,34 @@
+class Solution {
```
Comment (NidhiSingh26) [inline Leet Code/Reverse Nodes in k-Group:1]: Problem -  https://leetcode.com/problems/reverse-nodes-in-k-group/

**Comment by NidhiSingh26** [comment]
I have updated the file @mr-ayush-agrawal

**Comment by mr-ayush-agrawal** [comment]
Do mention it as comment as this is not a good practice to put some statement in the code.

File: Leet Code/Reverse Nodes in k-Group
Comment (NidhiSingh26) [inline Leet Code/Reverse Nodes in k-Group:1]: 25. Reverse Nodes in k-Group](https://leetcode.com/problems/reverse-nodes-in-k-group/)

Given the head of a linked list, reverse the nodes of the list k at a time, and return the modified list.

k is a positive integer and is less than or equal to the length of the linked list. If the number of nodes is not a multiple of k then left-out nodes, in the end, should remain as it is.

You may not alter the values in the list's nodes, only nodes themselves may be changed.

File: Leet Code/Median of two sorted arrays
Comment (NidhiSingh26) [inline Leet Code/Median of two sorted arrays:1]: 4. Median of Two Sorted Arrays](https://leetcode.com/problems/median-of-two-sorted-arrays/)

Given two sorted arrays nums1 and nums2 of size m and n respectively, return the median of the two sorted arrays.

The overall run time complexity should be O(log (m+n)).

**Pull request merged**